bitcoind.proxy

```diff
@@ -0,0 +1,79 @@
+upstream bitcoind {
+  # local server
+  server 127.0.0.1:8332;
+  # remote server
+  #server 192.168.0.22:18332
```

Comment:
The port would still be `8332` since/if this is for mainnet right?

Reply:
Thank you. I missed that. set the correct port to mainnet